src/Main.java

```diff
@@ -31,13 +33,17 @@ public static void main(String[] args) {
             sum3 = sum3 + payment3[i];
         }
         double averagePayment = (double)sum3 / weekInMonth;
-        System.out.printf("Средняя сумма трат за месяц составила "+ averagePayment + " рублей.");
+        System.out.println("Средняя сумма трат за месяц составила "+ averagePayment + " рублей.");
         // Задание 4
         System.out.println("Задание 4");
         char[] reverseFullName = {'n', 'a', 'v', 'I', ' ', 'v', 'o', 'n', 'a', 'v', 'I'};
-
-        for (int i = 0; i < reverseFullName.length; i++) {
-
+        char[] fullName = new char[11];
```

Comment:
Лучше все же длину массива не хардкодить, а взять из свойства
```suggestion
        char[] fullName = new char[reverseFullName.length];
```